recipes/pyslalib/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pyslalib" %}
+{% set version = "1.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pySLALIB-{{ version }}.tar.gz
+  sha256: 21d0e1ae6fc73ade9c90ec9e956b2da33163383388cc84e5ec58994bf66c8b35
+
+build:
+  number: 0
```

It looks like the upstream setup.py just doesn't support Windows at all causing the build to not actually compile any of the sources. If you care you'll need to patch it else I'd suggest just skipping it:
```suggestion
  skip: true  # [win]
  number: 0
```